### PR TITLE
Add option to provide custom stylesheets

### PR DESCRIPTION
**Pull request by bumi**
the CSS will be scoped to the widget

### CODE REVIEW

File: src/containers/widget/index.js [hunk 1]
```diff
@@ -210,6 +210,7 @@ Widget.defaultProps = {
   isModalOpen: false,
   invoiceDetails: null,
   fetchingInvoiceState: false,
+  styles: "",
```
Comment (reznord) [inline src/containers/widget/index.js:213]: question: styles will be a string by default?

**Review by reznord — APPROVED** [review]
LGTM

File: src/containers/widget/index.js (hunk 1 above)
Comment (bumi) [reply src/containers/widget/index.js:213]: I thought I do it similar to the other props we have. 
from the server we either get nothing (the `styles` key will not be in the response) or a string with styles

Comment (reznord) [reply src/containers/widget/index.js:213]: ok.

**Pull request merged**